recipes/geoplot/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+conda upgrade -y -q gdal
```

Comment:
Let's see how our CI behave when you remove it. I will help you fix any future errors.